src/PackageAnalyzer.jl

```diff
@@ -263,6 +263,9 @@ end
 # or they may ask for the dev version, which again we treat the same way.
 #
 # They may ask for a specific version, which we treat like release.
+"""
+Abstract type for package source
```

Comment:
I'm not much familiar with the inside of this package. Is it enough for the docstring? Or, should I include the above comments in the docstring?

Reply:
I think it would be good to have a little more info, like
```suggestion
    abstract type PkgSource
    
Represents the installed version of a package, e.g. a release from a registry, or a `Pkg.add`'d package, or a `Pkg.dev`'d package.
```